pkg/minikube/sshutil/sshutil.go

```diff
@@ -121,7 +122,12 @@ func Transfer(reader io.Reader, readerLen int, remotedir, filename string, perm
 		fmt.Fprint(w, "\x00")
 	}()
 
+	// Lets don't assume that we have sudo on every platform.
 	scpcmd := fmt.Sprintf("sudo scp -t %s", remotedir)
+	if runtime.GOOS == "windows" {
```

Comment:
This is actually running inside the guest VM, which in our case is always linux. Did this cause any specific issues for you?